sdcm/mgmt/cli.py

```diff
@@ -156,7 +156,7 @@ def status(self) -> str:
         # The manager will sometimes retry a task a few times if it's defined this way, and so in the case of
         # a failure in the task the manager can present the task's status as 'ERROR (#/4)'
         tmp = str_status.split()
-        if ' '.join(tmp[0:1]) == 'ERROR (4/4)':
+        if ' '.join(tmp) == 'ERROR (4/4)':
```

Comment:
This is not correct, time to time status has something after `ERROR (4/4)`, at least used to have
For example it could be `ERROR (4/4) Some reason to fail`.
Code that was before would see no difference in such case.
While your code will `return TaskStatus.ERROR_FINAL` only if there is nothing after `ERROR (4/4)`

Reply:
I don't recall seeing such message, but I changed the condition to be ready for that and added a comment.